docs/usage/storage_backends.rst

```diff
@@ -0,0 +1,39 @@
+..
+    This file is part of Invenio.
+    Copyright (C) 2015-2019 CERN.
+
+    Invenio is free software; you can redistribute it and/or modify it
+    under the terms of the MIT License; see LICENSE file for more details.
+
+Storage Backends
+----------------
+In order to get started we need to setup and configure a storage backend.
```

Comment:
I would in general first explain concepts. Why do we need to get started with a storage? What is a Storage backend? Why this abstraction?
https://docs.aws.amazon.com/AmazonS3/latest/dev/Introduction.html
https://cloud.google.com/storage/docs/storage-classes

I would mention something like that Invenio provides a default storage that is local storage using standard python lib (PyFSFileStorage). It defines an interface to create your own storage adapter. For example, Invenio-XRootD provides XRootD integration.